lib/puppet/type/acl/ace.rb

```diff
@@ -308,14 +390,23 @@ def keys
       to_hash.keys
     end
 
+    # Returns values of instance fields
+    #
+    # @return [Array] Values of fields from hash
     def values
       to_hash.values
     end
 
+    # Returns value of specified key in instance fields hash
+    #
+    # @return [Object] Value of field
```

Comment:
Since to_hash is using string keys I think the input param here also has to be a string?